js/tests/unit/scrollspy.spec.js

```diff
@@ -101,33 +131,27 @@ describe('ScrollSpy', () => {
         '    </div>',
         '  </div>',
         '  <div id="scrollspy-example" style="height: 100px; overflow: auto;">',
-        '    <div style="height: 200px;">',
-        '      <h4 id="masthead">Overview</h4>',
-        '      <p style="height: 200px;"></p>',
-        '    </div>',
-        '    <div style="height: 200px;">',
-        '      <h4 id="detail">Detail</h4>',
-        '      <p style="height: 200px;"></p>',
-        '    </div>',
+        '    <div style="height: 200px;" id="masthead">Overview</div>',
+        '    <div style="height: 200px;" id="detail">Detail</div>',
         '  </div>',
         '</div>'
       ].join('')
 
       const scrollSpyEl = fixtureEl.querySelector('#scrollspy-example')
       const rootEl = fixtureEl.querySelector('#root')
       const scrollSpy = new ScrollSpy(scrollSpyEl, {
-        target: 'ss-target'
+        target: '#ss-target'
```

Comment:
Is this still covered?

Reply:
if user put it without selector, is not. Can be 'tweaked' to support bad habits like this

Reply:
> Is this still covered?

Reply:
@XhmikosR it is covered now 😉